cabal.project.ci

```diff
@@ -0,0 +1,4 @@
+packages: .
+
+package hscli
+  ghc-options: -Werror
```

Comment:
This prevents the build from succeeding when there are warnings. I'd like to do something similar with `hlint` as well